code/style.css

```diff
@@ -1,4 +1,285 @@
+    margin-right: 16px;
+}
+
+header a {
+    margin-left: 24px;
+}
+
+.hamburger {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+    width: 30px;
+    height: 30px;
+    position: fixed;
+    top: 40px;
+    right: 40px;
+}
+
+.hamburger div {
+    width: 30px;
+    height: 5px;
+    background: #000000;
+    border-radius: 5px;
+    margin-right: 30px;
+    margin-left: 20px;
+}
+
+.hamburger :hover {
+    background: greenyellow;
+}
+
+
+/* Code that changes the order and appearance of the header below */
+
+
+/* smartphone */
+
+@media only screen and (max-width: 767px) {
+    header a {
+        display: none;
+    }
+    header {
+        justify-content: center;
+    }
+    header .logo {
+        flex-grow: 0;
+    }
+}
+
+
+/* tablet */
+
+@media only screen and (min-width: 768px) and (max-width: 992px) {
+    .logo--text {
+        display: none;
+    }
+    .logo {
+        display: flex;
+        justify-content: space-between
+    }
+    /* I am so trying to figuere out why the logo dosen't go the way I need it to so I took it away*/
+    .nav1 {
+        order: 1;
+    }
+    .nav2 {
+        order: 2;
+    }
+    .nav3 {
+        order: 4;
+    }
+    .nav4 {
+        order: 5;
+    }
+    header {
+        justify-content: space-between;
+    }
+}
+
+
+/* =========== HOVER AND STUFF =========== */
+
+a {
+    color: #000000;
+    text-decoration: none;
+    transition: 0.15s;
+    box-shadow: 0px 4px 0px #1732cc;
+}
+
+a:hover {
+    color: #f3b94e;
+    box-shadow: 0px 8px 0px #f19443;
+}
+
+h1,
+h2 {
+    font-family: 'Playfair Display', serif;
+    font-weight: 800;
+    margin: 0;
+}
+
+h1 {
+    font-size: 42px;
+    line-height: 56px;
+    margin-bottom: 16px;
+}
+
+h2 {
+    font-size: 24px;
+    line-height: 32px;
+    margin-top: 8px;
+    /* padding-bottom: 16px; */
+}
+
+.redlink {
+    font-family: 'PT Sans', sans-serif;
+    ;
+    font-weight: 800;
+    color: #000;
+    text-decoration: none;
+}
+
+p {
+    font-style: normal;
+    font-weight: normal;
+    font-size: 18px;
+    line-height: 30px;
+    color: rgba(0, 0, 0, 0.87);
+    margin: 0;
+    margin-bottom: 16px;
+}
+
+.credits {
+    font-size: 14px;
+    color: rgba(0, 0, 0, 0.20);
+    font-family: 'PT Sans', sans-serif;
+    font-weight: 400;
+}
+
+
+/* ======CONTENT =======*/
+
+.grid-container {
+    display: grid;
+    grid-gap: 48px 20px;
+    margin-bottom: 120px;
+}
+
+
+/* smartphone */
+
+@media only screen and (max-width: 767px) {
+    .grid-container {
+        grid-template-columns: repeat(1, 1fr);
+    }
+    h1 {
+        font-size: 24px;
+        line-height: 36px;
+        margin-bottom: 16px;
+    }
+}
+
+
+/* tablet */
+
+@media only screen and (min-width: 768px) {
+    .grid-container {
+        grid-template-columns: repeat(2, 1fr);
+    }
+}
+
+
+/* desktop */
+
+@media only screen and (min-width: 992px) {
+    .grid-container {
+        grid-template-columns: repeat(4, 1fr);
+    }
+}
```

Comment:
Impressive that you managed with both CSS Grid and flexbox! Just make sure you change the class name in the HTML file to correspond with this!